src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function notification(notifClass) {
+  const notif = document.createElement('div');
+
+  notif.className = notifClass;
+
+  notif.textContent = (notifClass === 'error-message')
+    ? 'Promise was rejected!'
+    : 'Promise was resolved!';
+
+  logo.parentNode.append(notif);
+}
+
+logo.addEventListener('click', () => {
+  return new Promise(resolve => {
+    resolve('message');
+  })
+    .then(notClass => notification(notClass));
+});
+
+setTimeout(() => {
+  // eslint-disable-next-line promise/param-names
+  return new Promise(reject => reject('error-message'))
+    .then(notClass => notification(notClass));
```

Comment:
```suggestion
  return new Promise((res, rej) => rej('error-message'))
    .catch
```
Второй аргумент этой функции завершает промис с ошибкой, тебе надо использовать его